modules/ha/pages/remove-failed-node.adoc

```diff
@@ -0,0 +1,69 @@
+= Removal of Failed Nodes
+:description: This page describes the procedure to remove a failed node.
+
+If a node fails in a highly available (HA) cluster (replication factor > 1), you can remove the failed node from the cluster while keeping all your data intact.
+After removal, use the xref:cluster-resizing:expand-a-cluster.adoc[cluster expansion] feature to restore your cluster to its original size.
+Restoring is important as node removal does not redistribute the data in your cluster.
+
+You should only consider removing a node when the node has failed.
+Removing a node from a cluster does not redistribute the data in the cluster.
+If all nodes are working in a cluster, and you want to reduce the size of the cluster, xref:cluster-resizing:shrink-a-cluster.adoc[shrink the cluster] instead.
+
+
+== Remove a single node
+If only a single node fails in a HA cluster, you can always remove the node without incurring any data loss.
+
+=== Prerequisites
+* Your cluster should have a replication factor greater than 1.
+* Service status in your TigerGraph cluster needs to meet the following requirements:
+You can check service status by running xref:system-management:management-with-gadmin.adoc#_gadmin_status[`gadmin status`].
++
+--
+** At least 1 `CTRL`(Controller) service is in `RUNNING` status
+** All `ZK`, `ETCD` and `EXE` services except those on the node being removed are in `RUNNING` status.
+--
+If the above services are down, and you cannot restart them, please contact mailto:support@tigergraph.com[TigerGraph Support].
```

Comment:
In case one nodes down and you want to start services on other nodes, use `gadmin start all --ignore-error`